pkg/service/elbv2.go

```diff
@@ -45,6 +45,7 @@ func findInstanceInTargetGroup(elbClient elbv2iface.ELBV2API, arn, instanceID st
 
 	target, err := elbClient.DescribeTargetHealth(input)
 	if err != nil {
+	        log.Infof("Failed while finding instance %v in all target groups: %v", instanceID, err.Error())
```

Comment:
I think this message should be specific for a target group as this function looks for an instance ID in a specific target group with `arn`, so maybe it should be:
```
log.Infof("failed finding instance %v in target group %v: %v", instanceID, arn, err.Error())
```
```

Reply:
```suggestion
	        log.Infof("failed finding instance %v in target group %v: %v", instanceID, arn, err.Error())
```